easybuild/easyconfigs/b/berkeley_upc/berkeley_upc-2.20.2-GCC-4.9.2.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+easyblock = 'ConfigureMake'
+
+name = 'berkeley_upc'
```

Comment:
maybe use `UPC`? any reason why you didn't use that? are there other UPCs out there?